src/Reflection/ReflectionClass.php

```diff
@@ -361,26 +362,26 @@ private function createMethodsFromTrait(ReflectionMethod $method): array
     }
 
     /** @return list<ReflectionMethod> */
-    private function getParentMethods(): array
+    private function getParentMethods(AlreadyVisitedClasses $classNameStack): array
```

Comment:
I'd like to quickly rename those parameters to `$alreadyVisitedClasses` if that's OK